src/main/src/jetty/database/gogoduck-testing/cars.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO cars_timeseries_url VALUES ('CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_World_monthly.nc','2016-04-27 02:29:34');
+INSERT INTO cars_timeseries_url VALUES ('CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_Australia_weekly.nc','2016-04-27 02:29:34');
```

Comment:
Weekly and monthly aren't really aggregate-able - perhaps just use the weekly one or have separate layers for weekly and monthly  - the processing should be the same though so its probably not required.